test/files/test_ros/srv/ServiceWithStringConstant.srv

```diff
@@ -0,0 +1,6 @@
+string EXAMPLE= here "#comments" are ignored, and leading and trailing whitespace removed
```

Comment:
To make it more clear I would propose to change the wording to something like
```suggestion
string EXAMPLE= here "#comments" are part of the string constant, and leading and trailing whitespace removed
int32 INTCONST=42  # Here comments should be ignored
```

Reply:
If you want to make it more clear for users I would suggest changing it [in the docs](http://wiki.ros.org/msg#Constants) rather than in a test that probably no one is going to look at unless it fails.
I agree with you that this would be a better description but I don't think it matters in this context.